Userland/Applications/Presenter/PresenterWidget.cpp

```diff
@@ -79,6 +96,7 @@ void PresenterWidget::set_file(StringView file_name)
     } else {
         m_current_presentation = presentation.release_value();
         window()->set_title(DeprecatedString::formatted(title_template, m_current_presentation->title(), m_current_presentation->author()));
+        m_current_presentation->set_color_scheme(m_current_presentation->background_color(), m_current_presentation->foreground_color());
```

Comment:
That makes no sense at all; the presentation should handle its color scheme internally as it can load it while it is being created.